extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -133,6 +136,8 @@ export class AmpAnalytics extends AMP.BaseElement {
       this.consentPromise_ = userNotificationManagerFor(this.win)
           .then(service => service.get(this.consentNotificationId_));
     }
+
+    this.scopingEle_ = this.element.getAttribute('scoped');
```

Comment:
Was trying to store the extension element tagName. Then found analytics parent can be any non AMP element. I'll remove it.